spec/cypress/support/step_definitions/common-navigation.js

```diff
@@ -3,14 +3,24 @@ import { Given, When, Then } from "cypress-cucumber-preprocessor/steps";
 Given("I am logged in as a {string}", (user) => cy.login(user));
 Given("I am logged in as an {string}", (user) => cy.login(user));
 
-Given("scenario {string} has been ran", (scenario) => cy.appScenario(scenario));
+Given("scenario {string} has been run", (scenario) => cy.appScenario(scenario));
 
 const pagePaths = {
   cookie: "/cookies",
   start: "/",
-  "admin listing": "/admin/administrators",
+  "admin index": "/admin/administrators",
   "admin creation": "/admin/administrators/new",
   "admin confirm creation": "/admin/administrators/new/confirm",
+  "delivery partner index": "/admin/suppliers",
+  "choose new delivery partner name":
+    "/admin/suppliers/new/delivery-partner/choose-name",
+  "choose new delivery partner lead providers":
+    "/admin/suppliers/new/delivery-partner/choose-lps",
+  "choose delivery partner cohorts":
+    "/admin/suppliers/new/delivery-partner/choose-cohorts",
+  "new delivery partner review": "/admin/suppliers/new/delivery-partner/review",
+  "delivery partner edit": /\/delivery-partners\/.*\/edit/,
+  "delivery partner delete": /\/delivery-partners\/.*\/delete/,
```

Comment:
Uh oh, double lines in json make me sad. Do we want to bump the char limit or look into some commonisation for long urls?

Reply:
I'd be very against changing the char limit in prettier as it isn't a limit, it's a target - it will also affect function definitions and other object declarations

I also think it looks fine! (also, this isn't json)

Reply:
FWIW, the ruby line length is longer than the prettier one. I prefer *slightly* longer lines than this personally